test/setup_helpers.sql

```diff
@@ -0,0 +1,32 @@
+
+----------------------------------------------------
+-- So we can know that an action that was specified in the charts actually exists in the database.
+create or replace function function_exists(schema text, fname text) returns bool as
+$$ BEGIN
+      IF NOT EXISTS (SELECT * FROM information_schema.routines WHERE routine_name = fname) -- TODO how constraint per schema?
+                                                                                          -- TODO you can also constraint by type if you feel like it
+      THEN           RETURN FALSE;
+      ELSE           RETURN TRUE;
+      END IF;
+    END
+$$ language plpgsql strict;
```

Comment:
We could potentially simplify this to
```sql
create or replace function function_exists(schema text, fname text) returns bool as
$$
  select exists (select 1 from information_schema.routines where routine_name = fname);
$$ language sql strict;
```

The extra made a bit confused at first